src/handlers/guilds/GUILD_DELETE.ts

```diff
@@ -20,30 +35,15 @@ export async function handleGuildDelete(
     }
   });
 
-  cacheHandlers.forEach("members", async (member) => {
+  cacheHandlers.forEach("members", (member) => {
     if (!member.guilds.has(payload.id)) return;
 
     member.guilds.delete(payload.id);
 
     if (!member.guilds.size) {
-      await cacheHandlers.delete("members", member.id);
-      return;
+      return cacheHandlers.delete("members", member.id);
     }
 
-    await cacheHandlers.set("members", member.id, member);
+    cacheHandlers.set("members", member.id, member);
```

Comment:
why noawait

Reply:
The purpose of the await here was for stack tracing not really for awaiting

Reply:
Ahh